### DISCUSSION TITLE
福州大学（旗山校区）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#19] [University]
> 回答来自群友，我仅负责搬运

211，双一流，非 985。
~~（话说真的会有群友报考福州大学吗喵？）~~

**提示**
 - 福州大学（旗山校区）分为教学区与生活区，生活区下分 5 个子区域。
 - 教学区与生活区距离一条马路（人行横道）
 - 步行通勤可能需要 10\~20 分钟

**宿舍是上床下桌吗？**
 - 是。（生活 1 区为 6 人间，其余为 4 人间）

**教室和宿舍有没有空调？**
 - 有喵～

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远？**
 - 有

**有早自习、晚自习吗？**
 - 无早自习
 - 多数学院对大一学生有强制晚自习要求，时间 19:00 - 21:00（或 21:30）
 - （据说有些学院大二的学生也要晚自习喵？）

**有晨跑吗？**
 - 无

**每学期跑步打卡的要求是多少公里，可以骑车吗？**
 - 学校无强制要求，但有此类活动可以加综测（早安福大）。
 - （当然，早安福大活动是不能骑车参加的喵～）

**寒暑假放多久，每年小学期有多长**
 - 30-60天，本科无小学期

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
 - 状况复杂喵～
 - 食堂本身有合作商家，这些商家有外送服务
 - 食堂的外送服务，疫情前可以送到寝室楼门，现在只能送到寝室楼下
 - 校外的似乎不能进生活区，要去偏僻的栏杆取

**学校附近有地铁站吗**
 - 有，但是在教学区东门

**宿舍楼有洗衣机吗**
 - 不一定，可能需要自行购买
 - （如果自行购买）通常要自己拿排插供电，至少 3 米。

**校园网怎么样**
 - 校园网由中国联通提供服务，需要本地联通手机卡，无 IPv6
 - 宿舍区可拉三大运营商宽带（但是有可能因为光纤段子不够而拉不了）
 - 教学楼是 CERNET

**每天断电断网吗，几点开始断**
 - 无断电断网

**食堂价格贵吗，会吃出异物吗**
 - 还算可以喵……？ （8-13元能吃饱，甚至可以再少一些）

**洗澡热水供应时间**
 - 全天（前提是记得用热水器烧水）

**校园内可以骑电瓶车吗，电池在哪能充电**
 - 可以
 - 宿舍楼下有充电桩，微信控制。

**宿舍限电情况**
 - 基本不限电
 - 吹风机等电器用完要收起来（至少断电），否则可能被扣分（可能是综测）

**通宵自习有去处吗**
 - 公共教学楼处有专门的考研自习室、讨论吧，似乎是 24 小时开放的。
 - 但座位似乎不多
 - 图书馆、教室情况尚不明确（卷不动了喵 QAQ）

**大一能带电脑吗**
 - “原则上不能。”
 - 实际情况因学院而异。不过大多数情况下，只要检查的人来宿舍时你没在打游戏，就不会出事。

**学校里面用什么卡，饭堂怎样消费**
 - 有校园卡（福大一卡通），支持 NFC，可以充值，主要用途是刷宿舍楼门禁和食堂消费
 - 有学生卡（但没有什么存在感喵？）
 - 食堂可使用一卡通，但也支持支付宝和微信支付（没有 Paypal，差评喵～）

**学校会给学生发银行卡吗**
 - 会。
 - 银行卡用于学费和住宿费缴交。奖助学金、校内比赛奖金等也会发放至学生银行卡。

**学校的超市怎么样**
 - 价格与品质和附近永辉超市持平（不计促销活动）

**学校的收发快递政策怎么样**
 - 快递接收：需要定点收取（包括京东物流）。部分物流公司的快递需要提前下载“菜鸟”App。
 - 快递发送：生活区快递点可以发件，也可以电话预约上门/到楼下
 - 生活区有两处快递点。一处大快递点位于生活区三区，一处邮政限定快递点位于生活区二区。

**学校里面的共享单车数目与种类如何**
 - 种类主要有 3 种（含电动助力车）
 - 共享单车：青橘；共享电单车：安安出行、哈罗出行
 - 数目：至少够用（

**现阶段学校的门禁情况如何**
 - 需要在“今日校园”上申请通行码，方可离校。
 - 多数当天往返的申请可秒通过。
 - 实际上，你甚至可以不申请而离校，但回校需要通行码。（导致你因为没有通行码而被拒之门外）

**宿舍晚上查寝吗，封寝吗，晚归能回去吗？**
 - 有查寝，严格程度视学院（或辅导员）而定
 - 23:00 前需回宿舍楼，封寝状况尚不明确。

**补充**
 - 必修课有补考，但所有选修课均无补考（只能重修，含专业选修）
 - 电动班车一次 1 元，多数情况下一辆最多搭乘 16-17 人

**Comment by KevlarKanou** [comment]
表单已提交
洗澡热水供应时间：独立卫浴的独立热水器，没有供应时间这回事
通宵自习有去处吗：基本没有。公共教学楼22:30熄灯。图书馆21:00关闭阅览室，23:00关闭自习室
大一能带电脑吗：各学院情况差异巨大
宿舍晚上查寝吗，封寝吗，晚归能回去吗？：各学院规定差异巨大
学校里面用什么卡，饭堂怎样消费：其实基本没人用一卡通，大多数人用微信和支付宝。不过每个档口都能用各种支付方式（说 Paypal 的也太婆罗门了吧）
学校里面的共享单车数目与种类如何：只剩青桔单车了（数量相当大），2021年9月起所有共享电动车退出旗山校区
宿舍楼有洗衣机吗：上一届宿舍的学长一般都把洗衣机卖了……

**Reply by KevlarKanou** [reply]
欢迎报考福大工科（CS，EE，电气），文科建议就别看福建的学校了，因为既然你都在看福大了你应该也考不上厦大……

**Comment by jihuayu** [comment]
补充一些问题里没有的内容：
旗山校区是福大的主校区，绝大部分的学生会在这里，部分学院大一新生会在其他校区。
宿舍大部分4人间，极其小部分6人间。
宿舍无打热水的地方，宿舍可以使用电热水壶，宿舍线路大概够3-4个电热水壶同时使用（没想到吧，这么大

> 必修课有补考，但所有选修课均无补考（只能重修，含专业选修）
> **补充，实践类必修也无补考。**

学生管理总体比较官僚（虽然所有大学都这样），具体看各自辅导员和学院。
偶尔会强行要求学生去参加一些没人去的活动（凑人头）。

上下学期间学校交通比较混乱，同学骑电动车非常**飘逸**。
计算机类学科的竞赛比较强。**但是，竞赛和大部分同学无关，刨除竞赛和实验室其他的学习氛围挺差的，需要自己努力。**
计算机的培养计划下届会改，终于不用面对傻逼培养计划了。
计算机 10% 向上的保研率。

## 建议
单从计算机类专业来说，是合格的中流 211，不能做到给垃圾镀金，想打 ACM 可以过来，毕业证省内认可度较高，其他看自己努力程度。
## 最后
### **别报人工智能实验班**
